packages/cli/src/commands/link/pollParams.js

```diff
@@ -5,12 +5,13 @@
  * LICENSE file in the root directory of this source tree.
  *
  * @format
+ * @flow
  */
 
 import inquirer from 'inquirer';
 
-export default questions =>
-  new Promise((resolve, reject) => {
+export default (questions: any) =>
```

Comment:
I think there's a type for that in `config`. Check `project.params`

Reply:
We do, it's `any` :p. Can import it though